src/storage-ors-preview/azext_storage_ors_preview/_help.py

```diff
@@ -0,0 +1,114 @@
+    text: az storage account ors-policy update -g ResourceGroupName -n StorageAccountName --source-account newSourceAccount --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8"
+  - name: Update source storage account in Object Replication Service Policy Rule.
+    text: az storage account ors-policy update -g ResourceGroupName -n StorageAccountName --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8" --rule-id "78746d86-d3b7-4397-a99c-0837e6741332" --sourceContainer newSourceContainer
+  - name: Update Object Replication Service Policy for storage account.
+    text: az storage account ors-policy update -g ResourceGroupName -n StorageAccountName -p @policy.json
+"""
+
+helps['storage account ors-policy rule'] = """
+type: group
+short-summary: Manage Object Replication Service Policy Rules.
+"""
+
+helps['storage account ors-policy rule add'] = """
+type: command
+short-summary: Add rule to the specified Object Replication Service Policy.
+examples:
+  - name: Add rule to the specified Object Replication Service Policy.
+    text: az storage account ors-policy rule add -g ResourceGroupName -n StorageAccountName --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8" -d destContainer -s srcContainer
+"""
+
+helps['storage account ors-policy rule list'] = """
+type: command
+short-summary: List all the rules in the specified Object Replication Service Policy.
+examples:
+  - name: List all the rules in the specified Object Replication Service Policy.
+    text: az storage account ors-policy rule list -g ResourceGroupName -n StorageAccountName --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8"
+"""
+
+helps['storage account ors-policy rule remove'] = """
+type: command
+short-summary: Remove the specified rule from the specified Object Replication Service Policy.
+examples:
+  - name: Remove the specified rule from the specified Object Replication Service Policy.
+    text: az storage account ors-policy rule remove -g ResourceGroupName -n StorageAccountName --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8" --rule-id "78746d86-d3b7-4397-a99c-0837e6741332"
+"""
+
+helps['storage account ors-policy rule show'] = """
+type: command
+short-summary: Show the properties of specified rule in Object Replication Service Policy.
+examples:
+  - name: Show the properties of specified rule in Object Replication Service Policy.
+    text: az storage account ors-policy rule show -g ResourceGroupName -n StorageAccountName --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8" --rule-id "78746d86-d3b7-4397-a99c-0837e6741332"
+"""
+
+helps['storage account ors-policy rule update'] = """
+type: command
+short-summary: Update rule properties to Object Replication Service Policy.
+examples:
+  - name: Update rule properties to Object Replication Service Policy.
+    text: az storage account ors-policy rule remove -g ResourceGroupName -n StorageAccountName --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8" --rule-id "78746d86-d3b7-4397-a99c-0837e6741332" --prefix-match blobA blobB
```

Comment:
Shall we replace this with dummy vaule?

Reply:
This id is not a valid one. In this way it is fine now.